src/card.js

```diff
@@ -19,8 +19,45 @@ const useStyles = makeStyles({
   },
 });
 
-function SingleCard({ name, author, gameLink, numberOfLikes }) {
+
+function SingleCard({ name, author, gameLink, numberOfLikes, creatorId, gridId }) {
```

Comment:
Optional but advised: use conventional method where the parameter is called props.

Reply:
passing properties this way is pretty popular too